integration-tests/tests/pages/revision.page.ts

```diff
@@ -0,0 +1,186 @@
+import { expect, type Page } from '@playwright/test';
+
+/**
+ * Page object for the sequence revision page.
+ * Handles revising existing sequences with updated metadata and/or sequence data.
+ */
+export class RevisionPage {
```

Comment:
This has a bunch of unused methods. Should we delete them? Or do you want to keep them for future migrations?